packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -189,52 +271,81 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
         /**
          * _Assumes users has already sent funds to their ICA, until #9193
          * @param {CosmosValidatorAddress} validator
-         * @param {Amount<'nat'>} ertpAmount
+         * @param {AmountArg} amount
          */
-        async delegate(validator, ertpAmount) {
-          trace('delegate', validator, ertpAmount);
-
-          // FIXME brand handling and amount scaling #9211
-          trace('TODO: handle brand', ertpAmount);
-          const amount = {
-            amount: String(ertpAmount.value),
-            denom: this.state.bondDenom,
-          };
-
-          const account = this.facets.helper.owned();
-          const delegatorAddress = this.state.chainAddress.address;
+        async delegate(validator, amount) {
+          trace('delegate', validator, amount);
+          const { helper } = this.facets;
+          const { chainAddress } = this.state;
 
-          const result = await E(account).executeEncodedTx([
+          // FIXME get values from proposal or args
```

Comment:
FIXME when? do we plan to get values from proposal? which issue is that?

Reply:
This comment can be removed. The original intent was for this to be fixed in #9211. It seems we should just proceed with `ChainAmount`'s for now.